docs/external-template-creation.md

```diff
@@ -84,3 +73,50 @@ export default {
 ```
 
 Once you've completed this step, your new sticker should now appear with the other stickers appended to the bottom of the shapes panel if the `enableStickers` feature flag is enabled.
+
+### Inserting stickers (SVGs) into a story
+
+To add stickers to your template story, go to the `Experiments` page (in the left-hand sidebar under the `Stories` section) and check the `Enable Stickers` checkbox. 
+
+Once turned on, all available stickers in the codebase should be available for selection under our base shapes in the shapes library panel.
+
+### Get the story JSON
+
+To get the JSON representation of a story in the editor:
+
+1. In the editor, open the story.
+2. Open `Chrome DevTools > Network`.
+3. In the editor, click the "Save draft" button. 
+4. You should see a POST XHR with JSON in the request payload. In that payload JSON, find and right-click the `story_data` field and click `Copy value`. 
+5. The story JSON should now be copied to your clipboard and ready to paste into a new file.
+
+**Note**: `Command+Shift+Option+J / Control+Shift+Alt+J` in the editor opens up a dialog you can copy/paste story JSON. However, this is missing some necessary fields for versioning.
+
+### Adding story JSON to the codebase as a new template
+
+Once you have the story JSON, several code changes are needed to add it to the list of default templates in the editor.
+
+1. In [`packages/templates/src/raw/`](https://github.com/google/web-stories-wp/tree/main/packages/templates/src/raw), commit your template's story JSON as a new file e.g. `<template_name>.json`.
```

Comment:
I think we also need to comb through the json and update all media srcs to follow:
```
"__WEB_STORIES_TEMPLATE_BASE_URL__/images/templates/..."
```

Reply:
Yes. There's a Jest test that complains if you forget it though.

Reply:
Yea that test is great. has definitely saved me a couple times

Reply:
Added a substep and an example.